platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/user/configuration/AGWorkflowUserConfiguration.sqml

```diff
@@ -0,0 +1,12 @@
+PACKAGE com.softicar.platform.workflow.module.workflow.user.configuration
+
+IMPORT com.softicar.platform.core.module.user.AGUser
+
+TABLE AGWorkflowUserConfiguration "Workflow.WorkflowUserConfiguration" {
+	
+	PK AGUser user
+	Boolean notify = TRUE 
```

Comment:
This might be too unspecific: Notify... _about what?_ or _upon what?_ or _how?_
I suggesto to rename this to `emailNotifications`